README.md

```diff
@@ -143,6 +143,7 @@ If the `Solution type` is `OPTIMAL`, it means that the solver has determined thi
 ### 6. Beautifying the schedule
 
 ```bash
+$ export TZ=Europe/London
```

Comment:
How about adding this to the `.env` file (or `.env.dist` in this case)? One is pretty much required to rely on it to use the scheduler anyway.